<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Chiffrage/Déchiffrage</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QFrame" name="frame_7">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="pushAddCrypto">
                <property name="text">
                 <string>Ajouter Fichier(s)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushRemoveAllCrypto">
                <property name="text">
                 <string>Enlever Tout</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboDirection">
                <property name="currentText">
                 <string>Chiffrage</string>
                </property>
                <item>
                 <property name="text">
                  <string>Chiffrage</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Déchiffrage</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushStartJob">
                <property name="text">
                 <string>Lancer</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="fileListViewCrypto">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="showGrid">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="1">
            <widget class="QLineEdit" name="linePassConfirm">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Passphrase :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_Confirm">
             <property name="text">
              <string>Confirmation :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="linePassEncrypt">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Calculs des Hash</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPlainTextEdit" name="plainTextHash"/>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="pushAddHash">
                <property name="text">
                 <string>Ajouter Fichier(s)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushRemoveAllHash">
                <property name="text">
                 <string>Enlever Tout</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboHash">
                <property name="currentText">
                 <string notr="true">Adler32</string>
                </property>
                <item>
                 <property name="text">
                  <string notr="true">Adler32</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">CRC24</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">CRC32</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">MD4</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">MD5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">RIPEMD-160</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">SHA-1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">SHA-224</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">SHA-256</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">SHA-384</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">SHA-256</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">SHA-512</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">SHA-3</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">SHAKE-128</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">SHAKE-256</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">Skein-512</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">Tiger</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">Whirlpool</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">Keccak-1600</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">GOST-34.11</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushStartHash">
                <property name="text">
                 <string>Lancer</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>150</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="fileListViewHash"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_log">
       <attribute name="title">
        <string>Logs de sortie</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushClearLog">
             <property name="text">
              <string>Tout Effacer</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="plainTextLog">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fic&amp;hier</string>
    </property>
    <addaction name="actionConfiguration"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuInfos">
    <property name="title">
     <string>Infos</string>
    </property>
    <addaction name="actionA_propos_de_Arsenic"/>
    <addaction name="actionA_propos_de_Botan"/>
    <addaction name="actionA_Propos_de_Argon2"/>
    <addaction name="actionA_Propos_de_Qt"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuInfos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionAbout_Arsenic">
   <property name="text">
    <string>About Arsenic</string>
   </property>
   <property name="toolTip">
    <string>About Arsenic</string>
   </property>
  </action>
  <action name="actionA_propos_de_Arsenic">
   <property name="text">
    <string>&amp;A Propos de Arsenic</string>
   </property>
   <property name="toolTip">
    <string>A Propos de Arsenic</string>
   </property>
  </action>
  <action name="actionA_propos_de_Botan">
   <property name="text">
    <string>A &amp;Propos de Botan</string>
   </property>
  </action>
  <action name="actionA_Propos_de_Argon2">
   <property name="text">
    <string>A Propos &amp;de Argon2</string>
   </property>
  </action>
  <action name="actionA_Propos_de_Qt">
   <property name="text">
    <string>A Propos de &amp;Qt</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>&amp;Configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="5"/>
 <resources/>
 <connections/>
</ui>
